### PR TITLE
Rick anderson/atsushina iis request restrict

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@atsushina I brought #552 here, you'll get the commit credit once it merges.

[Internal review URL](https://review.docs.microsoft.com/en-us/IIS/troubleshoot/request-restrictions?branch=pr-en-us-554)

### CODE REVIEW

File: iis/TOC.yml [hunk 1]
```diff
@@ -792,7 +792,9 @@
       items: 
         - name: Troubleshooting Problems with Microsoft Web Platform Installer
           href: troubleshoot/web-platform-installer-issues/troubleshooting-problems-with-microsoft-web-platform-installer.md
-- name: Extensions
+    - name: IIS Request Restrictions on Windows Client OS
+          href: troubleshoot/request-restrictions.md
```
Comment (Rick-Anderson) [inline iis/TOC.yml:796]:
```suggestion
        href: troubleshoot/request-restrictions.md
```

File: iis/TOC.yml [hunk 2]
```diff
@@ -792,7 +792,9 @@
       items: 
         - name: Troubleshooting Problems with Microsoft Web Platform Installer
           href: troubleshoot/web-platform-installer-issues/troubleshooting-problems-with-microsoft-web-platform-installer.md
-- name: Extensions
+    - name: IIS Request Restrictions on Windows Client OS
+        href: troubleshoot/request-restrictions.md
```
Comment (Rick-Anderson) [inline iis/TOC.yml:796]:
```suggestion
    - name: IIS Request Restrictions on Windows Client OS
       href: troubleshoot/request-restrictions.md
```

File: iis/TOC.yml [hunk 3]
```diff
@@ -792,7 +792,9 @@
       items: 
         - name: Troubleshooting Problems with Microsoft Web Platform Installer
           href: troubleshoot/web-platform-installer-issues/troubleshooting-problems-with-microsoft-web-platform-installer.md
-- name: Extensions
+    - name: IIS Request Restrictions on Windows Client OS
+       href: troubleshoot/request-restrictions.md
```
Comment (Rick-Anderson) [inline iis/TOC.yml:796]:
```suggestion
    - name: IIS Request Restrictions on Windows Client OS
      href: troubleshoot/request-restrictions.md
```

File: iis/TOC.yml [hunk 4]
```diff
@@ -792,7 +792,15 @@
...
-- name: Extensions
+    - name: Web Platform Installer Issues 2
+      items: 
+        - name: Troubleshooting Problems with Microsoft Web Platform Installer
+          href: troubleshoot/web-platform-installer-issues/troubleshooting-problems-with-microsoft-web-platform-installer.md
+    - name: Windows Client OS Request Restriction
+      items:
+        - name: IIS Request Restrictions on Windows Client OS
+          href: troubleshoot/request-restrictions.md
+ - name: Extensions
```
Comment (Rick-Anderson) [inline iis/TOC.yml:803]:
```suggestion
- name: Extensions
```

File: iis/TOC.yml [hunk 5]
```diff
@@ -792,7 +792,15 @@
       items: 
         - name: Troubleshooting Problems with Microsoft Web Platform Installer
           href: troubleshoot/web-platform-installer-issues/troubleshooting-problems-with-microsoft-web-platform-installer.md
-- name: Extensions
+    - name: Web Platform Installer Issues 2
+      items: 
+        - name: Troubleshooting Problems with Microsoft Web Platform Installer
+          href: troubleshoot/web-platform-installer-issues/troubleshooting-problems-with-microsoft-web-platform-installer.md
```
Comment (Rick-Anderson) [inline iis/TOC.yml:798]:
```suggestion
```

**Comment by Rick-Anderson** [comment]
@shirhatti  @atsushina please review & S&M and do a PR into live and #562 too

**Review by shirhatti — APPROVED** [review]

**Pull request merged**